apps/desktop/src/app/sync/sources/dropbox.ts

```diff
@@ -20,136 +18,77 @@ export const DropboxConnector: SourceConnectorDefinition = {
   factory: (data?: ConnectorSettings) => of(new DropboxImpl(data)),
 };
 
-const DROPBOX_VERIFIER_CODE_KEY = 'DROPBOX_VERIFIER_CODE';
-const DROPBOX_TOKEN_KEY = 'DROPBOX_TOKEN';
-const DROPBOX_REFRESH_TOKEN_KEY = 'DROPBOX_REFRESH_TOKEN';
+const TOKEN = 'DROPBOX_TOKEN';
 class DropboxImpl implements ISourceConnector {
   hasServerSideAuth = true;
   isExternal = false;
   resumable = true;
   private isAuthenticated: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
-  CLIENT_ID: string;
   dbx?: any;
   dbxAuth?: any;
 
   constructor(data?: ConnectorSettings) {
-    this.CLIENT_ID = data?.CLIENT_ID || '';
+    // eslint-disable-next-line no-empty-function
+  }
```

Comment:
why do you keep this empty constructor? doesn't seem to do anything and  I don't see any usage of data passed to it neither